extension/js/common/message-renderer.ts

```diff
@@ -461,10 +461,7 @@ export class MessageRenderer {
 
   private getMessageInfo = async (fullMsg: GmailRes.GmailMsg): Promise<MessageInfo> => {
     const sentDate = GmailParser.findHeader(fullMsg, 'date');
-    let sentDateStr = $('div.gK span[title]').attr('title');
-    if (!sentDateStr || isNaN(Date.parse(sentDateStr))) {
-      sentDateStr = sentDate ? new Date(sentDate).toLocaleString() : '';
-    }
+    const sentDateStr = sentDate ? new Date(sentDate).toLocaleString() : '';
```

Comment:
earlier we've used similar code, but got user feedback about incorrect format of date when printing messages, so we've switched to parsing message date directly from Gmail page (https://github.com/FlowCrypt/flowcrypt-browser/pull/5676)

so in this issue it'll be better to stick with webpage date parsing, but fix it to get correct message date